metrics/core/lib/src/data/model/build_data.dart

```diff
@@ -25,12 +27,40 @@ class BuildData extends Build implements DataModel {
           coverage: coverage,
         );
 
+  /// Unique id for multiple integrations
```

Comment:
```suggestion
  /// The unique identifier of this build. 
```
or 
```suggestion
  /// The unique identifier of a document representing this build.
```
What do you think?